css/style.css

```diff
@@ -439,3 +439,35 @@ ul {
   line-height: 20px;
   padding-right: 58px;
 }
+
+/* Block part */
+
+.part__line {
+  width: 1169px;
+  height: 1px;
+
+  opacity: 0.2;
+  border: 1px solid #342b2b;
+}
+
+.part__title {
+  font-family: Roboto Slab, sans-serif;
```

Comment:
don't need. It's by default
By the way, you have two font-family properties for the body selector
![image](https://user-images.githubusercontent.com/9587715/210445326-be30f488-6814-4581-b284-dc84f703f78c.png)

Reply:
+